--- /Users/paul/code/astor/examples/Math-issue-280/src/java/org/apache/commons/math/analysis/solvers/UnivariateRealSolverUtils.java
+++ /Users/paul/code/astor/examples/Math-issue-280/src/java/org/apache/commons/math/analysis/solvers/UnivariateRealSolverUtils.java
@@ -199,8 +199,8 @@
-			// FIX mm 280
-			// if (fa * fb > 0.0 ) {
-			throw new org.apache.commons.math.ConvergenceException(
-			"number of iterations={0}, maximum iterations={1}, " + 
-			("initial={2}, lower bound={3}, upper bound={4}, final a value={5}, " + 
-			"final b value={6}, f(a)={7}, f(b)={8}"), 
-			numIterations, maximumIterations, initial, 
-			lowerBound, upperBound, a, b, fa, fb);
+
+
+
+
+
+
+
+

