--- /Users/paul/code/astor/examples/Math-issue-288/src/main/java/org/apache/commons/math/optimization/linear/SimplexSolver.java
+++ /Users/paul/code/astor/examples/Math-issue-288/src/main/java/org/apache/commons/math/optimization/linear/SimplexSolver.java
@@ -63,2 +63,2 @@
-			if (org.apache.commons.math.util.MathUtils.compareTo(tableau.getEntry(0, i), minValue, epsilon) < 0) {
-				minValue = tableau.getEntry(0, i);
+			if (org.apache.commons.math.util.MathUtils.compareTo(tableau.getEntry(0, i), minValue, epsilon) < 0)
+
@@ -66 +66 @@
-			}
+

